docs/user/bots.rst

```diff
@@ -1421,9 +1436,12 @@ This does not affect URLs which already include the scheme.
 Shadowserver
 ^^^^^^^^^^^^
 
+In IntelMQ 2.3 a new Shadowserver JSON parser was added.
```

Comment:
Please rephrase this so that newcomers understand what is meant. I.e. there are parsers for the CSV format as well as for JSON (added in ..., for the API ...).

Reply:
Should be easier to understand now